transcom/mymove
# MB-9604 Bug yellow highlights for original and reweigh weights

## Description

This PR fixes a bug around handling of the yellow label indicators.

## Reviewer Notes

Is there anything you would like reviewers to give additional scrutiny?
- If a shipment has a billable weight set by the TIO, the yellow highlights won't show for the original weight or reweigh weight. Does this behavior make sense?

## Setup

Add any steps or code to run in this section to help others prepare to run your code:

```sh
make db_dev_e2e_populate
make server_run
make office_client_run
```

### Testing
Ephemeral deploy: https://office-milmove-pr-7488.mymove.sandbox.truss.coffee/

1. Log in as TIO in office app
2. Search for move code: `MULTRW`
3. Click on payment request to see details
4. Click on `Review weights` button
5. Click on `Review shipments`, there are 4 shipments that have varying weights to show the yellow indicators

## Code Review Verification Steps


## References

* [Jira story](https://dp3.atlassian.net/browse/MB-9604) for this change

## Screenshots

![image](https://user-images.githubusercontent.com/1522549/135693947-99371859-f0bf-492a-bc88-b01390a783fa.png)
